### PR TITLE
Fixed issue # 72 - Puppetboard showing "internal server 500"

### PR DESCRIPTION
**Pull request by pjamenaja**

### CODE REVIEW

File: templates/puppetdb-metrics-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <command>search</command>
+      </commands>
+    
+      <mbean>
+        <name>puppetlabs.puppetdb.population:name=num-nodes</name>
+        <attribute mode="read">Value</attribute>
+        <operation>objectName</operation>
+      </mbean>
+    </restrict>
+{{- end -}}
```
Comment (Xtigyro) [inline templates/puppetdb-metrics-configmap.yaml]: Please insert an empty line at the end of the file.

File: values.yaml [hunk 2]
```diff
@@ -348,6 +348,7 @@ puppetdb:
   #    cpu: 1000m
   ## Additional puppetdb container environment variables
   extraEnv: {}
+  enabledMetric: true
```
Comment (Xtigyro) [inline values.yaml]: Let's change this to:

```yaml
  metrics:
    enabled: false
```

File: templates/puppetdb-deployment.yaml [hunk 3]
```diff
@@ -87,6 +95,11 @@ spec:
         - name: puppetdb-storage
           persistentVolumeClaim:
             claimName: puppetdb-claim
+        {{- if .Values.puppetdb.enabledMetric }}
```
Comment (Xtigyro) [inline templates/puppetdb-deployment.yaml]: this will need to be changed respectively.

File: templates/puppetdb-deployment.yaml [hunk 4]
```diff
@@ -55,6 +55,14 @@ spec:
           volumeMounts:
             - name: puppetdb-storage
               mountPath: /opt/puppetlabs/server/data/puppetdb/certs
+            {{- if .Values.puppetdb.enabledMetric }}
```
Comment (Xtigyro) [inline templates/puppetdb-deployment.yaml]: this will need to be changed respectively.

**Review by Xtigyro — CHANGES_REQUESTED** [review]
Thanks for this contribution as well - very appreciated! 🙇

Two actions needed - could you please:
- Check the few change request comments I've left.
- Update the CHANGELOG and also bump the chart version in Chart.yaml?

CC: @underscorgan @slconley @Iristyle @scottcressi @mwaggett @nwolfe @adrienthebo @dhollinger @binford2k @alannab

**Comment by pjamenaja** [comment]
Will submit new changes after PR#71 is merged to master.

**Comment by Xtigyro** [comment]
Yes - makes sense.

File: values.yaml [hunk 5]
```diff
@@ -348,7 +348,8 @@ puppetdb:
   #    cpu: 1000m
   ## Additional puppetdb container environment variables
   extraEnv: {}
-
+  metrics:
+    enabled: false
```
Comment (Xtigyro) [inline values.yaml:353]: And if you could put a comment above those two lines with a little bit info what it enables. 

Also - in `README.md` could you please put info for those new parameters (there is such a section towards the end of the file).

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: values.yaml (hunk 5 above)
Comment (pjamenaja) [reply values.yaml:353]: Sure, I will.

**Comment by pjamenaja** [comment]
Added changes per suggestions above. Please let me know if there are more to change.

Thanks.

**Review by Xtigyro — APPROVED** [review]

**Comment by Xtigyro** [comment]
This is super great, @pjamenaja! Thank you once more! 🥇 💯 🎉 

CC: @underscorgan @slconley @Iristyle @scottcressi @mwaggett @nwolfe @adrienthebo @dhollinger @binford2k @alannab

**Pull request merged**